analytics/management/commands/populate_realms_db.py

```diff
@@ -0,0 +1,201 @@
+import itertools
+import os
+import random
+from collections import defaultdict
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Mapping, Sequence, Tuple
+
+import bmemcached
+import orjson
+from django.conf import settings
+from django.contrib.sessions.models import Session
+from django.core.files.base import File
+from django.core.management import call_command
+from django.core.management.base import BaseCommand, CommandParser
+from django.db import connection
+from django.db.models import F
+from django.db.models.signals import post_delete
+from django.utils.timezone import now as timezone_now
+from typing_extensions import override
+from corporate.lib.stripe import do_create_stripe_customer, update_or_create_stripe_customer
+from corporate.models import Customer, CustomerPlan, LicenseLedger
+
+from scripts.lib.zulip_tools import get_or_create_dev_uuid_var_path
+from zerver.actions.create_realm import do_create_realm
+from zerver.actions.custom_profile_fields import (
+    do_update_user_custom_profile_data_if_changed,
+    try_add_realm_custom_profile_field,
+    try_add_realm_default_custom_profile_field,
+)
+from zerver.actions.message_send import build_message_send_dict, do_send_messages
+from zerver.actions.realm_emoji import check_add_realm_emoji
+from zerver.actions.realm_linkifiers import do_add_linkifier
+from zerver.actions.scheduled_messages import check_schedule_message
+from zerver.actions.streams import bulk_add_subscriptions
+from zerver.actions.user_groups import create_user_group_in_database
+from zerver.actions.user_settings import do_change_user_setting
+from zerver.actions.users import do_change_user_role
+from zerver.lib.bulk_create import bulk_create_streams
+from zerver.lib.create_user import create_user, create_user_profile
```

Comment:
Could we use `do_create_user` instead? I generally prefer to do that where possible, just because these helper functions aren't intended to be externally used abstractions.

Reply:
Okay.